utils/scarb-macro-attributes/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, ItemFn};
+
+/// Inline macro helper.
+///
+/// This macro hides the conversion to stable ABI structs from the user.
+///
+/// # Safety
+/// Note that token stream deserialization may fail.
+#[proc_macro_attribute]
+pub fn inline_macro(_args: TokenStream, input: TokenStream) -> TokenStream {
+    let item: ItemFn = parse_macro_input!(input as ItemFn);
+    let item_name = &item.sig.ident;
+    let expanded = quote! {
+        use scarb_macro_interface::stable_abi::{StableProcMacroResult, StableTokenStream};
```

Comment:
I am unsure on this, but I'm afraid procmacros do not have same hygiene as `macro_rules!` and this `use` item may actually leak into `#item` and potentially cause problems in user code. Leaving as a topic to verify